src/gameplay/game.ts

```diff
@@ -21,6 +21,7 @@ import { avalonCards } from './cards/cards';
 import { avalonPhases, commonPhases } from './phases/phases';
 import { Card } from './cards/types';
 import { Role } from './roles/types';
+import {millisToStr} from "../util/time";
```

Comment:
Something is up with your prettier lol. Should have some spaces in the curly brackets.